guides/quickstart.md

```diff
@@ -0,0 +1,175 @@
+# Quickstart: How to configure and run the analyzer
+
+The code for this example is available in the `go-flow-levee/guides/quickstart` directory.
+
+## Context
+
+Suppose you have the following piece of code:
+
+`auth.go`
+```go
+package auth
+
+import "log"
+
+type Authentication struct {
+	Username string
+	Password string
+}
+
+func authenticate(auth Authentication) (*AuthenticationResponse, error) {
+	response, err := makeAuthenticatedRequest(auth)
+	if err != nil {
+		log.Printf("unable to make authenticated request: incorrect authentication? %v\n", auth)
+		return nil, err
+	}
+	return response, nil
+}
+```
+
+This code is an instance of [CWE-522](https://cwe.mitre.org/data/definitions/522.html), "Insufficiently Protected Credentials".
+The `Password` field on the `Authentication` struct contains credentials, which should not be allowed to reach logs.
+
+This example may seem contrived, but the analyzer has actually caught many similar cases in the wild.
+
+## Configuration
+
+Let's see how we can configure the analyzer to automatically detect this incorrect handling of credentials.
+
+In order to do its job, the analyzer needs 2 pieces of information:
+1. Which types hold sensitive data? (Sources)
+2. Which functions should not be called with sensitive data? (Sinks)
+
+Given this information, the analyzer will perform taint propagation to determine whether sensitive values can reach calls to functions that they shouldn't reach.
+
+### Configuring a Source type
+
+In taint propagation lingo, a value that contains data of interest is called a Source.
+In the current example, a Source is a value that contains sensitive data, e.g. a password.  
+
+There are 2 ways to define a Source type:
+1. Define a field tag and apply it to sensitive fields
+2. Describe a type using its name, the name of its package, and the relevant field(s)
```

Comment:
Nit: Markdown rendering ignores enumeration values and automatically renders `1. \\ 2. \\ `.  I believe the canonical approach is to use `1.` for all enumeration entries so that you don't have to manually re-enumerate them if an item is added or moved, and to let the renderer handle the actual incrementing of values.